blogs/network_events/index.md

```diff
@@ -0,0 +1,205 @@
+        matchLabels:
+          app: iperf3-client
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: iperf3-server-access-egress
+  namespace: iperf
+spec:
+  podSelector:
+    matchLabels:
+      app: iperf3-client
+  egress:
+  - ports:
+    - port: 5201
+      protocol: TCP
+  podSelector: {}
+  policyTypes:
+  - Egress
+```
+
+![ingress and egress network policy events](./images/ingr_egr_network_events.png)
+
+2. Admin Network Policies
+
+An Admin Network Policy is a high-level security policy designed to provide cluster administrators with
+more control over network traffic in Kubernetes environments
+eBPF agent can monitor:
+
+Allowed/Blocked Traffic: Detect whether packets are allowed or blocked based on admin network policies.
+
+Example of admin network policy and egress network policy to show multiple events use case :
+
+```yaml
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: allow-egress-iperf
+spec:
+  egress:
+  - action: Pass
+    to:
+    - namespaces: {}
+  priority: 10
+  subject:
+    namespaces:
+      matchLabels:
+        kubernetes.io/metadata.name: iperf
+```
+
+![admin network policy events](./images/admin_policy_network_events.png)
+
+3. Observe Egress Firewall Traffic
+
+For egress firewall monitoring, eBPF can:
+
+Trace Egress Traffic: Track outbound traffic and see whether it adheres to egress firewall rules.
+Detect Unauthorized Outbound Connections: You can flag outbound traffic that violates egress rules.
+
+Example includes three events, one for network policy, another for admin network policy and the third for egress firewall
+
+```yaml
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: default
+  namespace: iperf
+spec:
+  egress:
+  - to:
+      cidrSelector: 8.8.8.8/24
+    type: Allow
+  - to:
+      cidrSelector: 0.0.0.0/0
+    type: Deny
+```
+
+![egress firewall network events](./images/egress_fw_network_events.png)
+
+## Integration with Network Observability Stack
+
+- `eBPF Agent`:
+
+The agent will manage a new kprobe hook through a configuration option.
+It will also register with the OVN observability library, allowing eBPF-generated events to be enriched
+into meaningful strings.
+
+- `Agent User Space`:
+
+User space will handle the aggregation of events for processing.
+
+- `Flowlogs Pipeline`:
+
+Generated flows will be transmitted over gRPC to the FlowLog pipeline,
+where the flow content will be enriched with Kubernetes metadata.
+
+- `Loki storage and Console-plugin`:
+
+The enriched flows will then be sent to Loki storage, making them available for
+consumption by the NetObserv console plugin. The new events will be displayed in the console.
+
+- `New Filtering Capability`:
+
+A filter is introduced to filter flows based on network event enrichments.
+
+![network events building blocks](./images/network_events_design.drawio.png)
+
+## Configuration
+
+- Enable `OVNObservability` featureGate
```

Comment:
```suggestion
- Enable the `OVNObservability` feature in the `FeatureGate` custom resource.
```

Reply:
Just going off the language used here: https://docs.openshift.com/container-platform/4.17/nodes/clusters/nodes-cluster-enabling-features.html